### DISCUSSION TITLE
Using ProcessPoolExecutor in my task get an Exception: A process in the process pool was terminated abruptly while the future was running or pending.

### DISCUSSION THREAD

**Discussion by yjijn** [apache/airflow#29628] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

airflow version:2.1.3
python version:3.8.0

I use this function in my task and get the an exception
![image](https://user-images.githubusercontent.com/35212739/220051884-272cdcad-a6e0-4458-a2c7-cf5205ae4730.png)

![image](https://user-images.githubusercontent.com/35212739/220052346-4f602548-f3b9-4b58-969a-6304866be175.png)

### How to reproduce

run a task which using ProcessPoolExecutor

### Operating System

linux

### Deployment

Other

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
This is not airflow issue. This is your code (or someone else's so you need to check with whoever developed it) and raise your iss there. 

Converting it into a discussion in case you want to discuss it further and make it a troubleshooting of your code.

**Reply by potiuk** [reply]
Please in the future avoid opening issues which are not Airflow issues as issues - open discussions instead.
